syncopy/datatype/discrete_data.py

```diff
@@ -560,6 +589,10 @@ def __init__(self,
                          samplerate=samplerate,
                          dimord=dimord)
 
+        self._hdfFileDatasetProperties +=  ("waveform",)
```

Comment:
Would that not be better placed into `SpikeData` instead of the upstream general `DiscreteData` class?

Reply:
It _is_ in SpikeData.

Reply:
uh, yeah you're right.. sorry!